src/components/visits/VisitsDetailsComponent.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <section class="visitDetails">
+        <h3 class="visitDetails__header">Podsumowanie wydarzenia:</h3>
+        <article class="visitDetails__infoBox infoBox">
+            <h5 class="infoBox__label">Tytuł:</h5>
```

Comment:
jak wyzej

Reply:
we have to leave it cause prettier during prehook deleting white spaces

Reply:
ok, approve